index.md

```diff
@@ -2,8 +2,17 @@
 site: sandpaper::sandpaper_site
 ---
 
-This is a new lesson built with [The Carpentries Workbench][workbench]. 
+This lesson provides an overview of the fundamentals of imaging data that has been acquired through light microscopy experiments. This course is aimed at all researchers who are looking to work with light microscopy imaging data, or to refresh their knowledge on some of the key fundamentals.
 
+::::::::::::::::::::::::::::::::::::::::::  prereq
 
-[workbench]: https://carpentries.github.io/sandpaper-docs
+## Prerequisites
+
+- Some prior experience with acquiring/working with light microscopy data would be useful, but isn't necessary. This course is aimed at anyone who is looking to work with light microscopy imaging data in their research projects.
+
+- No python experience is necessary or assumed, but you will get more out of the course if you know some basics e.g. from the [Software Carpentry python course](https://swcarpentry.github.io/python-novice-inflammation/)
+
+- Please bring a laptop (Windows, Mac or Linux) to the lesson. Before the start of the course, please follow the setup instructions so that your laptop is ready to run the material in the lessons.
```

Comment:
Should we add link to setup instructions (even though it isn't populated yet), just so it's already there and we don't have to do it as part of another pull request?

Reply:
Good point! I'll add this